WTWSMS/decisions/WtWSMS_form_titles_decisions.txt

```diff
@@ -347,5 +370,37 @@ decisions = {
 			factor = 1
 		}
 	}
+
+	elevate_oriens = {
+		only_playable = yes
+		is_high_prio = yes
+		potential = {
+			has_landed_title = k_oriens
+			NOT = { tier = EMPEROR }
+		}
+		allow = {
+			independent = yes
+			prestige = 1000
+			wealth = 100
+		}
+		effect = {
+			primary_title = {
+			 	e_oriens = {
+					activate_title = { title = THIS status = yes } # Otherwise grant_title doesn't work
+					grant_title = PREVPREV
+					copy_title_laws = PREV
+				}
+			}
+			wealth - 100
+			prestige = 500
+		}
+			
+		revoke_allowed = {
+			always = no
+		}
+		ai_will_do = {
+			factor = 1
+		}
+	}
```

Comment:
There is an Empire of Oriens that was meant for that region. If, in the unlikely circumstance that Oriens somehow breaks free from Byzantium, the Proconsul of Oriens can form the Empire if he is prestigious enough and meets the requirements.

Also, you need to implement the change I made for the restoration of Israel. Right now, it requires a Jew to control all of Oriens, while in Vanilla, he only needs to control what is listed as Iudea and the duchy to irs right, which were the borders of Israel/Judea before Emperor Hadrian ended it all.